packages/react-component-playground/src/components/component-playground.jsx

```diff
@@ -11,8 +11,14 @@ var _ = require('lodash'),
     isSerializable = require('../lib/is-serializable.js').isSerializable,
     SplitPane = require('ubervu-react-split-pane'),
     localStorageLib = require('../lib/local-storage.js'),
+    CodeMirror = require('react-codemirror'),
     fuzzaldrinPlus = require('fuzzaldrin-plus');
 
+require('codemirror/lib/codemirror.css');
+require('codemirror/addon/fold/foldgutter.css');
+require('codemirror/theme/solarized.css');
```

Comment:
This seems to work:
![image](https://cloud.githubusercontent.com/assets/250750/18207128/7a43062e-7132-11e6-8f01-9d085dd46769.png)
![image](https://cloud.githubusercontent.com/assets/250750/18207133/7ee5a51a-7132-11e6-8b28-510d1061e518.png)
😎